### ISSUE TITLE
Platzbuchung auf bestimmte Plätze für einzelne Benutzer einschränken

### ISSUE THREAD

**Issue by mjmpeace** [tkrebs/ep3-bs#631]
Wir haben bei den Außenplätzen zwei Plätze, die auch fremde buchen können sollen. Kann man das irgendwie realisieren?

**Comment by RaphaelMHa** [comment]
Ja, im Fork von [Holger](https://github.com/hschiebold/ep3-bs) ist das über die folgende Checkbox möglich:
![grafik](https://github.com/tkrebs/ep3-bs/assets/97537168/dbb75558-24f3-40b3-9453-e4785839f512)

Zusätzlich muss man in den Einstellungen der Plätze, die von Gästen nicht benutzt werden dürfen, die Vereinszeit entsprechend einstellen:
![grafik](https://github.com/tkrebs/ep3-bs/assets/97537168/07b130ea-6454-4b64-b0fc-88033f2a10a8)

Die Preise kann man auch noch entsprechend dem Status einstellen:
![grafik](https://github.com/tkrebs/ep3-bs/assets/97537168/a10169f5-dd8a-405c-9a5a-6737b38277a7)

Paypal und Stripe funktionieren auch, sodass direkt bezahlt werden kann.
Wir haben bei uns 7 Außenplätze und 2 Hallenplätze. Gäste können bei uns nur Platz 5 und die Halle buchen.
